packages/stark-ui/tsconfig.json

```diff
@@ -12,9 +12,12 @@
       "@angular/common": ["../../node_modules/@angular/common"],
       "@angular/core": ["../../node_modules/@angular/core"],
       "@angular/material": ["../../node_modules/@angular/material"],
+      "@angular/material-moment-adapter": ["./node_modules/@angular/material-moment-adapter"],
```

Comment:
Is this path really needed even if this is part of the `node_modules` of `stark-ui` itself?

Reply:
As it is the case for `stark-core`, this path is needed for the compilation of the `testing` subpackage.